tests/unit_tests/test_resqpy.py

```diff
@@ -24,10 +24,8 @@ def test_all_imports():
 
 def test_version():
     # This is dynamically created when package is built
-    # If this fails when running tests locally, ensure you have installed the dev dependencies specified in setup.cfg
-    # In particular, try:  pip install setuptools_scm
     version_string = resqpy.__version__
 
     # Ensure version string is a PEP-440 compliant version
     version = Version(version_string)
-    assert version >= Version("1.1.1")
+    assert version >= Version("0.0.0")
```

Comment:
I'm a bit hesitant to disable this test: given that we've had some versioning issues in the past, it would give us some peace of mind to know the new system works.

I believe the version from Poetry-dynamic-versioning will only be present when the package is built (see [this thread](https://github.com/mtkennerly/poetry-dynamic-versioning/issues/25)). The CI tests are run against the built package, so it should be fine in that context.

However, we probably want to skip this test when running tests locally, as most devs will not have built the package.

One option for implementing this would be to create a custom `@requirebuild` decorator as detailed in [this thread](https://stackoverflow.com/questions/33084190/default-skip-test-unless-command-line-parameter-present-in-py-test), which disables the test unless the `--requirebuild` flag is passed to pytest.